src/main/java/com/arpnetworking/clusteraggregator/ClusterStatusCache.java

```diff
@@ -109,13 +128,51 @@ public Receive createReceive() {
                         }
                     }
                 })
+                .match(ShardRegion.ClusterShardingStats.class, shardingStats -> {
+                    LOGGER.debug()
+                            .setMessage("Received shard statistics")
+                            .addData("regionCount", shardingStats.getRegions().size())
+                            .log();
+                    final Map<String, Long> shardsPerAddress = Maps.newHashMap();
+                    final Map<String, Long> actorsPerAddress = Maps.newHashMap();
+                    for (final Map.Entry<Address, ShardRegion.ShardRegionStats> entry : shardingStats.getRegions().entrySet()) {
+                        final String address = entry.getKey().hostPort();
+                        for (final Map.Entry<String, Object> stats : entry.getValue().getStats().entrySet()) {
```

Comment:
Maybe values() instead of entrySet(). You never use the key

Reply:
Done